app/lib/common/themes/components/input_decoration_theme.dart

```diff
@@ -6,7 +6,7 @@ var inputDecorationTheme = InputDecorationTheme(
   filled: true,
   focusedBorder: OutlineInputBorder(
     borderSide: const BorderSide(
-      color: Colors.white70,
+      color: Colors.white54,
```

Comment:
we are changing the border color of all inputs? I think then we should also check other places and that this is really what we want...

Reply:
Yes checked with other inputs as well and felt that this should good so that's why changes this from common theme file instead of making change at component level.